OpenRA.Mods.Common/Effects/Missile.cs

```diff
@@ -147,7 +150,11 @@ public void Tick(World world)
 			var dist = targetPosition + offset - pos;
 			var desiredFacing = OpenRA.Traits.Util.GetFacing(dist, facing);
 			var desiredAltitude = targetPosition.Z;
-			var jammed = info.Jammable && world.ActorsWithTrait<JamsMissiles>().Any(JammedBy);
+
+			var jammed = false;
+			if (info.Jammable)
+				if (world.ActorsWithTrait<JamsMissiles>().Any(JammedBy))
```

Comment:
The idea was that the trait look-up should only be performed when the missile is jammable, wouldn't a combined if mean it does both checks?